### PR TITLE
Feature/feature 1

### PR DESCRIPTION
**Pull request by raghumarusu**
please review i have added feature1.1.txt file and merge it.

### CODE REVIEW

File: feature1.1.txt
Comment (raghumarusu) [inline feature1.1.txt:1]: please try with dictionary instead of map

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
 # llm
+Inference pipeline
+this is feature_1 by dev1
```
Comment (Raghumarusu442) [inline README.md:3]: please make change to this code create dictionary out of two lists

Comment (raghumarusu) [reply README.md:3]: changes done

**Comment by raghumarusu** [comment]
closing the pull request changes are satisfactory

**Pull request closed without merging**